fe/fe-core/src/main/java/com/starrocks/sql/optimizer/rule/transformation/materialization/MvUtils.java

```diff
@@ -1456,8 +1455,7 @@ public static Map<String, String> getPartitionProperties(MaterializedView materi
         partitionProperties.put("storage_medium", materializedView.getStorageMedium());
         String storageCooldownTime =
                 materializedView.getTableProperty().getProperties().get("storage_cooldown_time");
-        if (storageCooldownTime != null
-                && !storageCooldownTime.equals(String.valueOf(DataProperty.MAX_COOLDOWN_TIME_MS))) {
```

Comment:
`MAX_COOLDOWN_TIME_MS`(9999-12-31 23:59:59) is an invalid cool down time, why needs to update it?

Reply:
Seems fine. I found `OlapTable` does't check whether storage_cooldown_time property is MAX_COOLDOWN_TIME_MS or not either.

```
    public static DataProperty analyzeDataProperty(Map<String, String> properties,
                                                   DataProperty inferredDataProperty,
                                                   boolean isDefault)
```